packages/legend-graph/src/graphManager/GraphManagerState.ts

```diff
@@ -266,6 +267,9 @@ export class GraphManagerState extends BasicGraphManagerState {
       ...this.graph.ownClasses,
     ];
   }
+  get mainGraphClasses(): Class[] {
```

Comment:
maybe to keep in line with the rest we should use something like `usableAssociationPropertyClasses`